### DISCUSSION TITLE
Elastalert2 with Google Elastic Cloud

### DISCUSSION THREAD

**Discussion by mdiazcl** [jertel/elastalert2#565] [Q&A] [answered]
Hello!

Maybe i'm doing something wrong. I did my homework and checked Issues, Discussions and Google-fu and found nothing.
I installed the latest version of Elastalert2 from this repo, made a config file for elastalert and ran **elastalert-create-index** and got an **Unknown mimetype, Unable to deserialize: text/html** error.

BTW, I read the docs and I know that Cloud_id cannot be used.

On the config file (extract):
```
es_host: cluster-name.kb.us-east1.gcp.elastic-cloud.com
es_port: 9243
use_ssl: True
es_username: "elastalert"
es_password: "redacted"
```

I'am missing something?

**Comment by jertel** [comment, marked as answer]
Hello! I've never seen that error before. Attach the logs surrounding the Mimetype error so we can better understand what's going on.

**Reply by nsano-rururu** [reply]
The following problems?
https://github.com/Yelp/elastalert/issues/594
Unknown mimetype
>Have you tried upgrading or downgrading the Elasticsearch Python library? That's where this error is coming from. It seems that something (proxy, load balancer, ??) is converting the mimetype between ElastAlert and Elasticsearch. The actual content could be an error page or something similar.

Missing proxy configuration feature
https://github.com/Yelp/elastalert/issues/2017
>I have an issue while trying to query our ES behind a proxy, at start when the client try to get the ES version our proxy reject the request and a HTML page is returned by the proxy.
So ElastAlert crash on start with a "Unknow mime type html".

**Reply by nsano-rururu** [reply]
@mdiazcl

Do you know if a proxy is involved?

**Reply by nsano-rururu** [reply]
If you also need to set proxy settings when connecting to elasticsearch, you need to get a pull request to add functions.

**Reply by nsano-rururu** [reply]
Currently, Elastic Cloud may not be supported.

**Reply by nsano-rururu** [reply]
It may not be possible to reproduce the same phenomenon with Elastic Cloud without knowing what kind of settings are made.

**Reply by nsano-rururu** [reply]
If no information is provided, the investigation will remain uninvestigated.

**Reply by mdiazcl** [reply]
I feal insanely stupid... really; 
After days of trying, I figured out that my endpoint was all wrong... I was pointing to Kibana instead of ES

`cluster-name.kb.us-east1.gcp.elastic-cloud.com`

vs 

`cluster-name.es.us-east1.gcp.elastic-cloud.com`

Really easy to miss to be honest. But when you realize the "KB" vs the "ES" it makes you feel dumb. 
Sorry guys

**Reply by nsano-rururu** [reply]
I'm glad I noticed the mistake in the settings.
